Jenkinsfile

```diff
@@ -68,6 +69,90 @@ pipeline {
                 }
             }
         }
+        stage('Run performance regression testing') {
+            when {
+               not {
+                   branch 'master'
+               }
+            }
+            stages {
+                stage('Manual approval for regression testing') {
+                    steps {
+                        script {
+                            runBenchmarks = true
+                            try {
+                                timeout(time: 30, unit: 'SECONDS') {
+                                    input(message: 'Run optional performance regression tests?')
+                                }
+                            } catch (err) {
+                                runBenchmarks = false
+                            }
+                        }
+                    }
+                }
+                stage('Run benchmarks') {
+                    when {
+                        expression { return runBenchmarks == true }
+                    }
+                    steps {
+                        script {
+                            withCredentials([[
+                                $class: 'AmazonWebServicesCredentialsBinding',
+                                credentialsId: 'aws',
+                                accessKeyVariable: 'AWS_ACCESS_KEY_ID',
+                                secretKeyVariable: 'AWS_SECRET_ACCESS_KEY'
+                            ]]){
+                                // TODO: Modify Jenkins worker image to include terraform
+                                sh script: '''
+                                    # Install terraform, it should be moved to the image
+                                    if ! command -v terraform &> /dev/null
+                                    then
+                                        echo "Installing terraform"
+                                        curl -o terraform.zip "https://releases.hashicorp.com/terraform/0.14.2/terraform_0.14.2_linux_amd64.zip"
+                                        unzip terraform.zip
+                                        sudo mv terraform /usr/local/bin/
+                                        terraform --version
+                                    fi
+                                ''', label: 'Install terraform'
```

Comment:
This should be removed [once terraform is installed in the image](https://github.com/answerbook/jenkins/pull/156).